src/main/java/org/folio/oaipmh/helpers/GetOaiMetadataFormatsHelper.java

```diff
@@ -0,0 +1,142 @@
+package org.folio.oaipmh.helpers;
+
+import io.vertx.core.Context;
+import io.vertx.core.http.HttpMethod;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import javax.xml.bind.JAXBException;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+import org.apache.log4j.Logger;
+import org.folio.oaipmh.MetadataPrefix;
+import org.folio.oaipmh.Request;
+import org.folio.oaipmh.ResponseHelper;
+import org.folio.rest.tools.client.Response;
+import org.folio.rest.tools.client.interfaces.HttpClientInterface;
+import org.openarchives.oai._2.ListMetadataFormatsType;
+import org.openarchives.oai._2.OAIPMH;
+import org.openarchives.oai._2.OAIPMHerrorcodeType;
+import org.openarchives.oai._2.VerbType;
+
+public class GetOaiMetadataFormatsHelper extends AbstractHelper {
+
+  private static final Logger logger = Logger.getLogger(GetOaiMetadataFormatsHelper.class);
+
+  private final Context ctx;
+  private final HttpClientInterface httpClient;
+  private final Map<String, String> okapiHeaders;
+
+  public GetOaiMetadataFormatsHelper(HttpClientInterface httpClient, Context ctx,
+    Map<String, String> okapiHeaders) {
+    this.httpClient = httpClient;
+    this.okapiHeaders = okapiHeaders;
+    this.ctx = ctx;
+  }
+
+  @Override
+  public CompletableFuture<String> handle(Request request, Context ctx) {
```

Comment:
It's redundant to pass `Context` into the constructor as well as the handler method... Please choose one.